Tests/SymbolKitTests/SymbolGraph/SymbolGraphTests.swift

```diff
@@ -63,6 +63,19 @@ class SymbolGraphTests: XCTestCase {
         XCTAssertTrue(declaration.declarationFragments.contains(where: { fragment in
             fragment.kind == .externalParameter && fragment.spelling == "completionHandler"
         }))
+
+        // The async declaration should have been saved as an alternate declaration
+        let alternateDeclarations = try XCTUnwrap(symbol.alternateDeclarations)
+        XCTAssertEqual(alternateDeclarations.count, 1)
+        let alternate = try XCTUnwrap(alternateDeclarations.first)
```

Comment:
Nit:
Since we've checked the `count == 1`, maybe we can access it directly.
```suggestion
        let alternate = alternateDeclarations[0]
```